src/lib/djerba/plugins/supplement/body/plugin.py

```diff
@@ -63,6 +63,12 @@ def extract(self, config):
         else:
             report_signoff_date = wrapper.get_my_string(self.REPORT_SIGNOFF_DATE)
         self.check_assay_name(wrapper)
+       
+        if wrapper.get_my_attributes()[0] == "clinical":
+            include_signoffs = True
+        elif wrapper.get_my_attributes()[0] == "research":
+            include_signoffs = False
```

Comment:
Need a fallback in case it is neither clinical nor research

Could do `else: include_signoffs = False` and maybe raise a warning